src/nvim/api/vim.c

```diff
@@ -335,7 +335,7 @@ void nvim_input_mouse(String button, String action, String modifier,
     if (strequal(action.data, "down")) {
       code = KE_MOUSEUP;
     } else if (strequal(action.data, "up")) {
-      code = KE_MOUSEDOWN;
+      // pass
```

Comment:
assert(code == KE_MOUSEDOWN);

Reply:
That assertion is always true because of the outer-if condition.

Reply:
Yeah. This PVS warning is annoying, because if the code is rearranged later then a case may be missed. Unlikely for this case, I guess.